core/src/main/java/io/questdb/cairo/CairoEngine.java

```diff
@@ -185,7 +205,11 @@ public void unlockReaders(CharSequence tableName) {
     }
 
     public boolean releaseAllWriters() {
-        return writerPool.releaseAll();
+        boolean released = writerPool.releaseAll();
+        if (null != backupWriterPool) {
+        	return backupWriterPool.releaseAll() || released;
+        }
+        return released;
     }
```

Comment:
I decided to release the backup writer pool in this method rather than in its own method. It is not clear how useful the return value of this method is, it will be false if there were no writers or if we failed to release the writers within the deadline (infinite?). The method only seems to be used by tests in conjunction with the  releaseAllReaders, the return value seems to always be discarded. We may want to just have a single method to release all pools.